.eslintrc.js

```diff
@@ -25,9 +25,6 @@ module.exports = {
         '**/__tests__/*.{j,t}s?(x)',
```

Comment:
Remove this complete entry for `overrides`, also remove test entries from tsconfig.json file